cluster/test-deploy/gcp-logging/vars-origin.yaml

```diff
@@ -23,7 +23,43 @@ openshift_template_service_broker_namespaces: [ "openshift" ]
 ansible_service_broker_install: false
 openshift_enable_service_catalog: false
 template_service_broker_install: false
+
 openshift_logging_install_logging: true
+openshift_logging_es_nodeselector:
```

Comment:
The only thing that should be in vars-origin.yaml is standard variables across all profiles of the same type.  If these variables are part of the gcp-logging profile only, they should be in `vars.yaml`

Reply:
Makes sense, done